src/two-globals.js

```diff
@@ -0,0 +1,65 @@
+var count = 0;
+
+var TwoGlobals = {
```

Comment:
Same as with `EventTypes` is there a more generic name we can use here? Like just `globals` or you'll probably hate me for my ignorance of CS paradigms but like `namespace` or `singleton`?

Reply:
I'll rename it to `Globals`.